locales/en/dashboard.ftl

```diff
@@ -140,10 +141,10 @@ dashboard-top-banner-non-us-your-data-is-protected-description =
 
 dashboard-top-banner-monitor-more-cta = Monitor more emails
 
-# About Exposure Statuses Modal
+# About Exposure Indicators Modal
 
-modal-exposure-status-title = About exposure statuses
 modal-exposure-status-description-all = We search for exposures in all known data breaches.
   Your exposures will have one of the following statuses:
-modal-exposure-status-action-needed = <b>Action needed</b> means it is currently active and you need to take steps to fix it.
-modal-exposure-status-fixed = <b>Fixed</b> means the exposure has been resolved and there’s no action for you to take.
+modal-exposure-indicator-title = Exposure statuses
+modal-exposure-indicator-action-needed = Advanced or manual action is needed by you to complete an action.
```

Comment:
Sorry, realized this only when switching my brain to translator mode. "action needed to complete an action" doesn't make a lot of sense?